web/packages/teleport/src/Console/Console.tsx

```diff
@@ -44,6 +47,15 @@ export default function Console() {
   const documents = storeDocs.getDocuments();
   const activeDoc = documents.find(d => d.id === activeDocId);
   const hasSshSessions = storeDocs.getSshDocuments().length > 0;
+  const { attempt, run } = useAttempt();
+
+  function initUser() {
+    run(() => consoleCtx.initStoreUser());
+  }
+
+  React.useEffect(() => {
+    initUser();
+  }, []);
```

Comment:
i didn't see any other uses for `initUser`
```suggestion
  React.useEffect(() => {
     run(() => consoleCtx.initStoreUser());
  }, []);
```